gson-ktx/gson-parsing-ktx.md

```diff
@@ -0,0 +1,111 @@
+<h1 align="center">Awesome Android Extensions - <a href="https://chetangupta.net" target="_blank">AndroidBites</a>
+</h1>
+
+![awesome-android-extension-androidbites](./../androidbites_awesome_extension_banner.jpg)
```

Comment:
hi paths isn't correct can you please fix it

Reply:
@ch8n its already the correct path.